subprojects/docs/src/transforms/release-notes.gradle

```diff
@@ -1,7 +1,13 @@
 buildscript {
+
+    def mavenCentralUrl = project.rootProject.ext.repositoryMirrors?.get("mavencentral")
+
     repositories {
-        mavenCentral()
```

Comment:
Does the build still work if we remove the repositories block completely here? IIUC, the new `allproject.buildscript` would add the `gradle` mirror here which mirrors `jcenter` which mirrors `mavenCentral`?

Reply:
No it doesn't work if removing it directly. In my understanding, each script can configure its own `buildscript.repositores`. `allproject.buildscript` only configures the project's `build.gradle` script.

Reply:
I tried that and found that `project.buildscript` and `buildscript` in `release-note.gradle` get different `ScriptHandler`.

Reply:
Yes, `project.buildscript` only refers to the project build script's `buildscript {}` block. There is no way to inject into the equivalent of detached script plugins.